lib/vsc/utils/optcomplete.py

```diff
@@ -360,7 +361,7 @@ def print_usage_nousage (self, *args, **kwargs): # pylint: disable=unused-argume
     prev_interspersed = gparser.allow_interspersed_args  # save state to restore
     gparser.disable_interspersed_args()
 
-    cwords = os.environ.get('COMP_WORDS', '').split()
+    cwords = shlex.split(os.environ.get('COMP_WORDS', '').strip('() '))
```

Comment:
@JensTimmerman Can you explain this? Seems like this is beyond cleanup, and an actual bug fix instead?

Reply:
yep, tests fail when a directory in the path has spaces in it, so have to do this to make jenkins happy. Should I create separate pr? all of this cleanup is just to make jenkins happy...

Reply:
Maybe mention that in a comment to clarify?